packages/odf/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/body.tsx

```diff
@@ -517,7 +518,7 @@ export const LocalVolumeSetBody: React.FC<LocalVolumeSetBodyProps> = ({
               <SingleSelectDropdown
                 id="create-lvs-disk-size-unit-dropdown"
                 selectOptions={diskSizeUnitDropdownOptions}
-                selectedKey={diskSizeUnitOptions[state.diskSizeUnit]}
+                selectedKey={STORAGE_SIZE_UNIT_NAME_MAP[state.diskSizeUnit]}
```

Comment:
why not keep it as `diskSizeUnitOptions` only (not changing original variable) ?? this works too, right ??
```suggestion
                selectedKey={diskSizeUnitOptions[state.diskSizeUnit]}
```